projects/lights_control/inc/lights_can.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "can.h"
+#include "lights_events.h"
+
+#define LIGHTS_CAN_NUM_RX_HANDLERS 1
+
+typedef enum {
+  LIGHTS_CAN_ACTION_SIGNAL_RIGHT = 0,
+  LIGHTS_CAN_ACTION_SIGNAL_LEFT,
+  LIGHTS_CAN_ACTION_SIGNAL_HAZARD,
+  LIGHTS_CAN_ACTION_HORN,
+  LIGHTS_CAN_ACTION_HIGH_BEAMS,
+  LIGHTS_CAN_ACTION_LOW_BEAMS,
+  LIGHTS_CAN_ACTION_DRL,
+  LIGHTS_CAN_ACTION_BRAKES,
+  LIGHTS_CAN_ACTION_STROBE,
+  LIGHTS_CAN_ACTION_SYNC,
+  NUM_LIGHTS_CAN_ACTIONS_ID
```

Comment:
If this is your intended naming scheme, ID should be pluralized, not actions

Reply:
I've updated and separated different light actions.